lib/services/messenger/translators/front.ts

```diff
@@ -236,11 +247,12 @@ export class FrontTranslator extends TranslatorScaffold implements Translator {
 	/**
 	 * Converts a provided message object into instructions to create a message.
 	 * @param connectionDetails  Details of the connection to find things user ID.
+	 * @param metadataConfig     Configuration to use to encode metadata
 	 * @param message            object to analyse.
 	 * @returns                  Promise that resolves to emit instructions.
 	 */
 	private static createMessageIntoEmit(
-		connectionDetails: FrontConstructor, message: TransmitInformation
+		connectionDetails: FrontConstructor, metadataConfig: MetadataConfiguration, message: TransmitInformation,
```

Comment:
Comma again, am I missing something obvious here? There are many more functions with last parameter including a comma, I'm not going to point them all out. We certainly allow commas in the final entry in a structure, but not in a function.